dll/win32/comdlg32/lang/cdlg_Ro.rc

```diff
@@ -278,51 +278,51 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
  CONTROL "" , IDC_COLOR_RESULT, "STATIC",SS_SIMPLE|WS_TABSTOP|WS_GROUP, 152,124,40,26
- DEFPUSHBUTTON "Con&firmă",  IDOK,  4, 167, 50, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
- PUSHBUTTON "A&nulează", IDCANCEL, 58, 167, 50, 14, WS_GROUP | WS_TABSTOP
- PUSHBUTTON "&Manual…", pshHelp,100,166, 44, 14
- PUSHBUTTON "A&daugă culoarea personalizată",    IDC_COLOR_ADD /*1024*/, 152, 167, 144, 14, WS_GROUP | WS_TABSTOP
- PUSHBUTTON "&Definește colori personalizate >>", IDC_COLOR_DEFINE /*1025*/,   4, 149, 142, 14, WS_GROUP | WS_TABSTOP
- PUSHBUTTON "Color |  Sol&id", IDC_COLOR_RES, 300,200,4,14   /* just a dummy */
+ DEFPUSHBUTTON "OK",  IDOK,  4, 167, 50, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
+ PUSHBUTTON "Revocare", IDCANCEL, 58, 167, 50, 14, WS_GROUP | WS_TABSTOP
+ PUSHBUTTON "Aj&utor", pshHelp,100,166, 44, 14
+ PUSHBUTTON "&Adăugare la Culori particularizate",    IDC_COLOR_ADD /*1024*/, 152, 167, 144, 14, WS_GROUP | WS_TABSTOP
+ PUSHBUTTON "&Definire Culori particularizate >>", IDC_COLOR_DEFINE /*1025*/,   4, 149, 142, 14, WS_GROUP | WS_TABSTOP
+ PUSHBUTTON "Culoare |  Sol&id", IDC_COLOR_RES, 300,200,4,14   /* just a dummy */
 }
 
 
 FINDDLGORD DIALOG 36, 24, 276, 62
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Căutare"
+CAPTION "Găsire"
 FONT 8, "MS Shell Dlg"
 {
- LTEXT "C&aută:", -1, 4, 8, 52, 8
+ LTEXT "&De găsit:", -1, 4, 8, 52, 8
  EDITTEXT edt1, 57, 7, 148, 12, ES_AUTOHSCROLL | WS_BORDER | WS_GROUP | WS_TABSTOP
- CHECKBOX "D&oar cuvintele întregi", chx1, 4, 26, 140, 12, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
- CHECKBOX "&Distinge MAJ/min", chx2, 4, 42, 140, 12, BS_AUTOCHECKBOX | WS_TABSTOP
- GROUPBOX "Direcție:", grp1, 147, 21, 58, 38
- CONTROL "În s&us", rad1, "BUTTON", BS_AUTORADIOBUTTON | WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP, 151, 30, 48, 12
- CONTROL "În j&os", rad2, "BUTTON", BS_AUTORADIOBUTTON | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 151, 44, 48, 12
-
- DEFPUSHBUTTON "&Caută", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON
- PUSHBUTTON "A&nulează", IDCANCEL	 , 212, 24, 60, 14, WS_GROUP | WS_TABSTOP
- PUSHBUTTON "&Manual…", pshHelp	 , 212, 42, 60, 14, WS_GROUP | WS_TABSTOP
+ CHECKBOX "Potrivire numai &cuvinte complete", chx1, 4, 26, 140, 12, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
+ CHECKBOX "&Potrivire litere mari şi mici", chx2, 4, 42, 140, 12, BS_AUTOCHECKBOX | WS_TABSTOP
+ GROUPBOX "Orientare", grp1, 147, 21, 58, 38
+ CONTROL "în &sus", rad1, "BUTTON", BS_AUTORADIOBUTTON | WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP, 151, 30, 48, 12
+ CONTROL "în j&os", rad2, "BUTTON", BS_AUTORADIOBUTTON | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 151, 44, 48, 12
+
+ DEFPUSHBUTTON "&Următorul găsit", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON
+ PUSHBUTTON "Revocare", IDCANCEL	 , 212, 24, 60, 14, WS_GROUP | WS_TABSTOP
+ PUSHBUTTON "&Ajutor", pshHelp	 , 212, 42, 60, 14, WS_GROUP | WS_TABSTOP
```

Comment:
In most cases you used "Aj&utor" for the pshHelp (which exists several times in the file). But here you used "&Ajutor". Is that difference intended and correct?

Reply:
All of them must be harmonized. I also noticed this. In the beginning I thought I was wrong, but not. This is MSDN's decision.

![2024-04-30 00_26_42-Resource Hacker - comdlg32 dll mui](https://github.com/reactos/reactos/assets/113553683/f1390aa9-f3b6-43b5-ac4e-57434780b39c)
![2024-04-30 00_26_55-Resource Hacker - comdlg32 dll mui](https://github.com/reactos/reactos/assets/113553683/fe8657c6-a644-4060-a2e0-4d6c069d7b8f)

Reply:
Resolved.